org.eclipse.xtext.relengutil/pom.xml

```diff
@@ -24,4 +26,24 @@
 			<version>${project.version}</version>
 		</dependency>
 	</dependencies>
+
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>org.eclipse.tycho</groupId>
+				<artifactId>tycho-maven-plugin</artifactId>
```

Comment:
This plugin is useless: it's already enabled by the parent POM

Reply:
Copy and paste mistake